docs/management/rbac.md

```diff
@@ -0,0 +1,88 @@
+---
+title: "Role-Based Access Control (RBAC)"
```

Comment:
rename the `title` to `Console RBAC` to match with the title provided in the site. otherwise we will lose highlighting the selected nav item. title should be the same title provided in site side nav item.